crates/graphql/src/schema/query_root.rs

```diff
@@ -501,4 +505,43 @@ impl QueryRoot {
     fn denylist() -> Denylist {
         Denylist
     }
+
+    #[graphql(description = "Global summary statistics over indexed transactions.")]
+    fn stats_global(&self, context: &AppContext) -> FieldResult<GlobalStats> {
+        let conn = context.shared.db.get().context("failed to connect to db")?;
+
+        let i: u32 = 1000;
+
+        // TODO can/should these two queries be turned into async queries? If so, how do I get started on that?
+
+        // TODO: how do I convert this into UTC epoch seconds?
```

Comment:
What is the timezone of the `NaiveDateTime` returned for timestamp fields? I need this in order to correctly compute elapsed time.

Reply:
`NaiveDateTime`, as the name suggests, has no timezone.  It's up to you to convert it to a `DateTime<...>` for the right offset.  Generally data from the indexer is in UTC, but be sure to test that.